Removal of non-used imports

riha112 opened the pull request:
**Original issue:**
* https://github.com/scandipwa/scandipwa/issues/3054
_(as example, but there where and are many other issues)_

**Requires**
*  PR: https://github.com/scandipwa/scandipwa/pull/3081
* `@scandipwa/chunk-optimizer`

**In this PR**
* Clean up for non-needed imports after new plugin import

zans-laksa reviewed (changes requested):
1. The `m2-theme` plugin serves for the different purpose. Please put your code to a new plugin under `packages` directory.
2. How this will work on real projects when SPWA theme is located in the `node_modules`? What about style overrides? Also real projects doesn't have access to change the order of style chunks if needed.
3. So the next time when we encounter such issue, we should put a new style chunk or move existing one? What if we have a situation when components are overriding each other like Rock, Paper, Scissors?

riha112 commented:
@zans-laksa 
1. Created new PR for plugin `chunk-optimizer`: https://github.com/scandipwa/scandipwa/pull/3072, will update this PR after plugin is merged.
2. Correct, I forgot about the fact that the scandipwa is also located in node_modules, added exception for that.
3. In all cases I tested I could not break style as it was before with this setup... but if that happens I added support for dynamic imports _(more info in PR for plugin)_

zans-laksa reviewed (approved):
Code review passed.
